docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,243 @@
+
+🔲 Walk away with a introductory understanding of the *CIT* developer workflow.
+
+🔲 Feel confident in moving forward to our next recommended section: *[Cobalt Templating from Scratch](./3_NEW_TEMPLATE.md).*
 
 ## 2.3 Prerequisites
-< Enumerate any prerequisite walkthroughs that should be completed prior to this, and any technical prerequisites such as developer environment / tools >
 
-## 2.4 Walkthrough
-< Step-by-step instructions for completing this walkthrough >
\ No newline at end of file
+> NOTE: Previous "infrastructure as code" experience is not a prerequisite for completing the quick start guide.
+
+| Prereqs | Description |
+|----------|--------------|
+|Azure Subscription |[Azure Portal](https://portal.azure.com/) - This template needs to deploy infrastructure within an Azure subscription.|
+|Azure Service Principal|[Azure Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) - This template needs permissions to deploy infrastructure within an Azure subscription.|
+|Azure Storage Account|[Azure Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) - An account for tracking Terraform remote backend state. You can use our backend state setup [template](../infra/templates/backend-state-setup/README.md) to provision the storage resources.|
+|Azure CLI (`Latest`)|[Get started with Azure CLI](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli?view=azure-cli-latest) - The Azure CLI is a dependency of the Terraform templates used in Cobalt.|
+|GitHub Account|[Github](https://github.com/login) - An account for forking the Cobalt repo.|
+|Terraform `(0.12 +)`|[Terraform](https://www.terraform.io/downloads.html) - Download the appropriate version for setting up Terraform.|
+|Git Source Control|[Install git](https://git-scm.com/downloads).|
+|Terminal with bash shell|[WSL](https://code.visualstudio.com/docs/remote/wsl) or [Bash](https://git-scm.com/downloads) - The shell environment needed to follow along with the provided instructions. *In Windows, we recommend Visual Studio Code integrated WSL*.|
+---
+
+## 2.4 Run Cobalt's [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md)
+
+Below are the steps for running the [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md) from your terminal. Ensure that running this template achieves the quick start guide's final goal of deploying the following Azure cloud infrastructure before you can call this quick start guide finished:
+
+| Preview Steps | Final Goal |
+|----------|--------------|
+|![cobalt-'NickeManarin/Screen2Gif'](./screentogif-40sec-test.gif)| <image src="https://user-images.githubusercontent.com/10041279/66678697-149f1280-ec32-11e9-9bbb-c94a8111115b.png" width="1400"/> |
```

Comment:
This gif has a lot of extra clicks and searches through `ctrl + r`. I'm not sure what its trying to show. Perhaps we could re-record with the steps pre-copied and pasted in another window, so it is more to the point?

Reply:
Gif has been updated with most steps.